docs/getting_started.md

```diff
@@ -94,22 +111,43 @@ def main() -> None:
 
     # Instantiate CATX
     catx = CATX(
-        rng_key=subkey,
-        network_builder=MLPBuilder(),
+        custom_network=MyCustomNetwork,
         optimizer=optax.adam(learning_rate=0.01),
         discretization_parameter=8,
         bandwidth=1 / 8,
     )
 
     # Training loop
     costs_cumulative = []
-    for _ in range(1000):
+    for i in range(1000):
         obs = environment.get_new_observations()
         if obs is None:
             break
-        actions, probabilities = catx.sample(obs=obs, epsilon=0.05)
+
+        if i == 0:
+            network_extras = {"dropout_rate": 0.2}
+            state = catx.init(
+                obs=obs, epsilon=epsilon, key=key, network_extras=network_extras
+            )
+
+        state.network_extras["dropout_rate"] = 0.2
```

Comment:
Nitpicking here but maybe dropout should be 0 for sampling and 0.2 for learning no?